raiden/settings.py

```diff
@@ -46,4 +46,6 @@
 ETHERSCAN_API = 'https://{network}.etherscan.io/api?module=proxy&action={action}'
 
 RED_EYES_CONTRACT_VERSION = '0.4.0'
-DEVELOPMENT_CONTRACT_VERSION = '0.3._'
+DEVELOPMENT_CONTRACT_VERSION = '0.7.0'
```

Comment:
Just to be sure this is intended: This changes the development version to use the constrained version of the contracts.

Tests passing with this would mean that we never test having multiple token networks, which would be surprising.

Reply:
It was intended because i want to use the version of raiden-contracts which includes the UserDeposit contract... but i am fighting with the smoketest regarding the missing limits. I'll be looking at the contracts to see how i can do this.

Reply:
Fellback to 0.6.0